report_thesis/src/sections/background.tex

```diff
@@ -178,16 +183,118 @@ \subsubsection{Kernel PCA}
+
+$$
+f(x) = \frac{1}{M} \sum_{m=1}^{M} f_m(x),
+$$
+
+where $f_m(x)$ is the prediction of the $m$-th tree, and $M$ is the total number of trees.
+
+\gls{etr} extends the \gls{rf} model by introducing additional randomness in the tree-building process, specifically through random feature selection and random split points.
+While \gls{rf} uses bootstrap sampling and selects the best split from a random subset of features to create a set of diverse samples, \gls{etr} instead selects split points randomly within the chosen features, introducing additional randomness.
+This process results in even greater variability among the trees, aiming to reduce overfitting and improve the model's robustness.
+As a tradeoff, \gls{etr} is less interpretable than a single decision tree, as the added randomness can introduce more bias than \gls{rf}.
+However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
+
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+\gls{gbr} is an ensemble learning method that sequentially construct multiple decision trees, each one correcting the errors of the previous one, using gradient descent and boosting techniques.
+Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions.
+This ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+
+Initially, the prediction is set as $\hat{f}_0(\mathbf{x}) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set, where $\mathbf{x}$ represents the vector of input feature(s) and $y$ is the true value or target variable.
+The model is then iteratively improved over $B$ iterations, where $B$ is a hyperparameter controlling the total number of trees.
+With each iteration $b$ from $1$ to $B$, predictions are updated as:
+$$
+    \hat{f}^{(b)}(\mathbf{x}) = \hat{f}^{(b-1)}(\mathbf{x}) + \lambda \hat{f}_b(\mathbf{x}),
+$$
+where $\hat{f}_b(\mathbf{x})$ is the prediction of the $b$-th tree and $\lambda$ is the learning rate. Residuals are updated as:
+$$
+    r_i^{(b)} = y_i - \hat{f}^{(b-1)}(\mathbf{x}_i),
+$$
+Each tree is then trained on these updated residuals.
+The repetitive process of fitting a weak learner to predict the residuals and using its predictions to update the model gives us the final model:
+$$
+    \hat{f}(\mathbf{x}) = \sum_{b=1}^{B} \lambda \hat{f}_b(\mathbf{x})
+$$
+In the context of regression, gradient boosting aims to minimize the difference between the predicted values and the actual target values by fitting successive trees to the residuals.
+To minimize errors, gradient descent is used to iteratively update model parameters in the direction of the negative gradient of the loss function, thereby following the path of steepest descent~\cite{gradientLossFunction}.
```

Comment:
In the context of regression, gradient boosting aims to minimize prediction errors by fitting successive decision trees to the residuals of previous predictions. Each new tree corrects errors made by previous trees. Gradient descent is used to optimize the model parameters by updating them in the direction of the negative gradient of the loss function, helping the model minimize errors efficiently.

Reply:
I resolve these because they are handled by another PR.